src/devices/Bmx280/README.md

```diff
@@ -0,0 +1,33 @@
+# BMx280 Sensors
+
+## Summary
+
+BMx280 is a device family that read barometric pressure, altitude and temperature. SPI and I2C can be used to communicate with the device (only I2C implemented so far).
```

Comment:
Can we add to this readme a checkbox of supported protocols that show that the device supports both I2c and SPI where we only check the i2c box as that is the only one we have for now? Here is an example of where we already do that: [https://github.com/dotnet/iot/blob/e9529392e86b8e0e31e96ba1a5f83110ecf55639/src/devices/Ssd1306/README.md#connection-type](https://github.com/dotnet/iot/blob/e9529392e86b8e0e31e96ba1a5f83110ecf55639/src/devices/Ssd1306/README.md#connection-type)